tests/by-util/test_cp.rs

```diff
@@ -1392,15 +1392,14 @@ fn test_cp_target_file_dev_null() {
 fn test_cp_one_file_system() {
     use crate::common::util::AtPath;
     use walkdir::WalkDir;
-
```

Comment:
Why do you remove this line? Seems like a useful separator between imports and variable declarations.